code/messaging/build.gradle.kts

```diff
@@ -21,11 +21,7 @@ plugins {
 val mavenCoreVersion: String by project
 val mavenEdgeVersion: String by project
 val mavenEdgeIdentityVersion: String by project
-val toolingPreviewVersion = "1.7.4"
-val viewModelComposeVersion = "2.8.6"
-val runtimeComposeVersion = "2.8.6"
-val material3Version = "1.3.0"
-val mockkVersion = "1.13.13"
+val material3Version = "1.2.0"
```

Comment:
Snapshot tests are failing due to change in material3 version. Will update the baseline screenshots once this PR is merged.

Reply:
For future reference, would you be able to add a comment on why we cannot go any lower on the `material3` version (i.e what feature needs this).

Reply:
Just added a comment. Card component is marked @ExperimentalMaterial3Api in lower versions and had a few bugs that were fixed with v1.2.0